app/Domain/Subscriptions/Subscription.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Domain\Subscriptions;
+
+use Ramsey\Uuid\UuidInterface;
+
+final class Subscription
+{
+    public function __construct(
```

Comment:
Add integration type relation, see https://github.com/cultuurnet/publiq-platform/pull/2#discussion_r1011642286

Reply:
Added an `integrationType` to a `Subscription`